### ISSUE TITLE
Error retrieving String value from XLSX sheet using Cell.asString if values are boolean

### ISSUE THREAD

**Issue by cameron-hurd** [Coreoz/Windmill#4]
Attempting to retrieve a String value from an XLSX that has values like TRUE and FALSE throws the following:

```
java.lang.IllegalStateException: Cannot get a STRING value from a BOOLEAN cell

	at org.apache.poi.xssf.usermodel.XSSFCell.typeMismatch(XSSFCell.java:1077)
	at org.apache.poi.xssf.usermodel.XSSFCell.getRichStringCellValue(XSSFCell.java:395)
	at org.apache.poi.xssf.usermodel.XSSFCell.getRichStringCellValue(XSSFCell.java:71)
	at com.coreoz.windmill.imports.parsers.excel.ExcelCell.asString(ExcelCell.java:39)
	at com.coreoz.windmill.WindmillTest.parsingFunction(WindmillTest.java:232)
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
	at java.util.stream.SliceOps$1$1.accept(SliceOps.java:204)
	at java.util.stream.SliceOps$1$1.accept(SliceOps.java:204)
	at java.util.Spliterators$IteratorSpliterator.tryAdvance(Spliterators.java:1812)
	at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126)
	at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:499)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:486)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:566)
	at com.coreoz.windmill.WindmillTest.should_export_excel_data_starting_from_a_non_origin_point(WindmillTest.java:84)
```

**Comment by cameron-hurd** [comment]
Please see fix PR - https://github.com/Coreoz/Windmill/pull/5

**Comment by amanteaux** [comment]
Many thanks for this issue and your PR !

This is fixed in version 1.2.1 : https://github.com/Coreoz/Windmill/releases/tag/1.2.1
